developer-docs-site/docs/nodes/indexer-fullnode.md

```diff
@@ -16,54 +16,58 @@ The below installation steps are verified only on macOS with Apple silicon.
+  -  [`postgres` PostgreSQL server](https://www.postgresql.org/) - `brew install postgresql`
+  - [`diesel`](https://diesel.rs/) - `brew install diesel`
 
-- Install [Brew](https://brew.sh/).
-- Install Cargo and Rust via [Install Rust](https://www.rust-lang.org/tools/install).
-- Install [libpq](https://formulae.brew.sh/formula/libpq). This is a Postgres C API library. Make sure to perform all export commands after the installation.
-  - macOS: `brew install libpq`
-  - Linux: `apt install libpq`
-- Install [PostgreSQL](https://www.postgresql.org/):
-  - macOS: `brew install postgres`
-  - Linux: `apt install postgres`
-- Install [Diesel](https://diesel.rs/):
-`cargo install diesel_cli --no-default-features --features postgres`.
+TODO: Test and provide a single `brew install` command for everything after `cargo` here. Also find out if we need Cargo if they go the Docker route.
 
-### Setup
+## Set up database
 
 1. Start the PostgreSQL server: 
-   - macOS: `brew services start postgresql`
-   - Linux: `pg_ctl -D /opt/homebrew/var/postgres start`
-2. Run the following command to create a PostgreSQL user `postgres` (macOS commmand example below):
+   `brew services start postgresql`
+1. Ensure you can run `psql postgres` and then exit the prompt by entering: `\q`
+1. Create a PostgreSQL user `postgres` with the `createuser` command (find it with `which`):
    ```bash
-   /opt/homebrew/bin/createuser -s postgres
+   /path/to/createuser -s postgres
    ```
-3. Ensure you are able to do: `psql postgres `.
-4. Install the Diesel CLI: 
-    ```bash
-    cargo install diesel_cli --no-default-features --features postgres
-    ```
-5. Clone `aptos-core` repo:
+1. Clone `aptos-core` repository if you have not already:
     ```bash
     git clone https://github.com/aptos-labs/aptos-core.git
     ```
-6. `cd` into `aptos-core/crates/indexer` directory.
-7.  Run the command:
+1. Navigate (or `cd`) into `aptos-core/crates/indexer` directory.
+1.  Create the database schema:
     ```bash
     diesel migration run --database-url postgresql://localhost/postgres
     ```
-    This will create a database schema with the subdirectory `migrations` located in this `aptos-core/crates/indexer` directory.
-    - If for some reason this database is already being used, try a different database. For example: `DATABASE_URL=postgres://postgres@localhost:5432/indexer_v2 diesel database reset`
+    This will create a database schema with the subdirectory `migrations` located in this `aptos-core/crates/indexer` directory. If for some reason this database is already in use, try a different database. For example: `DATABASE_URL=postgres://postgres@localhost:5432/indexer_v2 diesel database reset`
+
+## Start fullnode indexer
+
+TODO: Determine if this should be done in the `aptos-core` checkout as indexer startup is looking for a `Cargo.toml` file:
+```
+➜  mainnet cargo run --bin aptos-node --features "indexer"  -- --config fullnode.yaml
```

Comment:
cargo run -p aptos-node --features "indexer" --release -- -f /path/to/fullnode.yaml